<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="tj2">
        <node name="tj2_bar_pipeline_py" pkg="tj2_bar_pipeline" type="tj2_bar_pipeline_node.py" output="screen" required="false">
            <param name="depth_topic" value="/camera/depth/image_rect_raw"/>
            <param name="info_topic" value="/camera/depth/camera_info"/>
            <param name="min_distance" value="0.3"/>
            <param name="max_distance" value="5.5"/>
            <param name="bar_z_lower_threshold" value="0.3"/>
            <param name="bar_z_upper_threshold" value="5.5"/>
            <param name="line_mask_width" value="1"/>
            <param name="contour_perimeter_threshold" value="200"/>
            <param name="hough_lines_threshold" value="100"/>
            <param name="line_angle_lower_threshold" value="-0.7854"/>
            <param name="line_angle_upper_threshold" value="0.7854"/>
        </node>
    </group>
</launch>
